server/chat-plugins/helptickets.js

```diff
@@ -758,6 +848,165 @@ const pages = {
 			buf += `</tbody></table></div>`;
 			return buf;
 		},
+		stats(query, user, connection) {
+			// view-help-stats-TABLE-YYYY-MM-COL
+			if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
+			this.title = 'Ticket Stats';
+			if (!this.can('lock')) return;
+
+			let table = query.shift() || '';
+			if (!['staff', 'tickets'].includes(table)) table = 'tickets';
+			let monthString = `${query.shift()}-${query.shift()}`;
```

Comment:
I think using a destructuring assignment to create some variables would be more readable

Reply:
You'll have to clarify what you want me to do here since I dont know what you mean.

Reply:
```ts
let [year, month] = query;
let monthString = `${year}-$month}`;
```